lsp-clients/n4js-vscode-extension/package.json

```diff
@@ -63,6 +63,24 @@
 					"command": "n4js.rebuild"
 				}
 			]
+		},
+		"configuration": {
+			"title": "N4JS",
+			"properties": {
+				"n4js.traceOutput": {
+					"type": "string",
+					"default": "off",
+					"enum": [
+						"off",
+						"verbose"
+					],
+					"enumDescriptions": [
+						"No tracing data is displayed.",
+						"Tracing data is displayed on the Output of N4JS language server"
```

Comment:
Output -> output

Reply:
Thanks, I changed the wording since I intended to refer to the Output View of VSCode.